src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -1227,6 +1230,8 @@ data class UpdatePlotObservationRequestPayload(
 data class UploadPlotPhotoRequestPayload(
     val gpsCoordinates: Point,
     val position: ObservationPlotPosition,
+    @Schema(description = "Type of observation plot photo. Defaulted to Plot")
```

Comment:
FYI, `@Schema` has a `defaultValue` attribute you can use to document the default.